waspc/data/Generator/templates/server/src/crud/_operations.ts

```diff
@@ -39,9 +50,7 @@ const entities = {
 // 1. We either use the default implementation of the operation...
 =}
 {=^ overrides.GetAll.isDefined =}
-type GetAllInput = {}
```

Comment:
For some reason, we built these types in two places: in the server and the in the SDK, but there was no need since the server can import the built types from the SDK. There is only one place where CRUD operation types are now defined.